src/Identity/Extensions.Core/src/PasswordHasher.cs

```diff
@@ -222,7 +242,7 @@ private static bool VerifyHashedPasswordV2(byte[] hashedPassword, string passwor
 
             // Hash the incoming password and verify it
             byte[] actualSubkey = KeyDerivation.Pbkdf2(password, salt, Pbkdf2Prf, Pbkdf2IterCount, Pbkdf2SubkeyLength);
-            return CryptographicOperations.FixedTimeEquals(actualSubkey, expectedSubkey);
+            return ByteArraysEqual(actualSubkey, expectedSubkey);
```

Comment:
```suggestion
#if NETSTANDARD2_0
            return ByteArraysEqual(actualSubkey, expectedSubkey);
#elif NETCOREAPP3_0
            return CryptographicOperations.FixedTimeEquals(actualSubkey, expectedSubkey);
#else
#error Update target frameworks
#endif
```

Let's not undo https://github.com/aspnet/AspNetCore/pull/9798